doc/source/rllib-algorithms.rst

```diff
@@ -405,7 +405,7 @@ Derivative-free
 
 Augmented Random Search (ARS)
 -----------------------------
-|tensorflow|
+|pytorch| |tensorflow|
```

Comment:
Don't forget to update the sigils in rllib-toc.rst

Reply:
+1 Added to toc as well.